src/tlo/methods/cardio_metabolic_disorders.py

```diff
@@ -208,6 +208,13 @@ class CardioMetabolicDisorders(Module):
                   'nc_risk_score': Property(Types.INT, 'score to represent number of risk conditions the person has')
                   }
 
+    @staticmethod
+    def is_next_test_due(current_date, date_of_last_test):
+        return (
+            pd.isnull(date_of_last_test)
+            or (current_date - date_of_last_test).days > DAYS_IN_YEAR / 2
+        )
+
```

Comment:
It was. Though the method itself is only actually called once (across the entire class & codebase), so we could even just resolve this with a variable assignment rather than a nested function too.